docs/howto/operate/add-aws-hub.md

```diff
@@ -0,0 +1,211 @@
+[kops](https://kops.sigs.k8s.io/getting_started/aws/)-based AWS
+cluster and manually deploy a new hub on top of it using our deployer tool.
+This is a preliminary but fully functional and manual process. Once
+[#381](https://github.com/2i2c-org/pilot-hubs/issues/381) is resolved, we should be able
+to automate the hub deployment process as we currently do with the GKE-based hubs.
+
+```{note}
+We will continue working toward a definitive one once we figured out some of the
+discussions outlined in [#431](https://github.com/2i2c-org/pilot-hubs/issues/431).
+```
+
+## Pre-requisites
+
+1. Follow the instructions outlined in
+[Set up and use the the deployment scripts locally](./manual-deploy.md) to set up the
+local environment and prepare `sops` to encrypt and decrypt files.
+
+2. Install the awscli tool (you can use pip or conda to install it in the environment)
+and configure it to use the provided AWS user credentials. Follow this
+[guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config)
+for a quick configuration process.
+
+```{note}
+The customer with AWS admin privileges should have created a user for you with full
+privileges. We will probably explore fine-graining the permissions actually needed
+in the short-term.
+```
+
+3. Export some helpful AWS environment variables (because "aws configure" doesn't export
+these environments vars for kops to use, so we do it manually).
+
+```bash
+export AWS_PROFILE=<cluster_name>
+export AWS_ACCESS_KEY_ID=$(aws configure get aws_access_key_id)
+export AWS_SECRET_ACCESS_KEY=$(aws configure get aws_secret_access_key)
+```
+
+## Create an AWS kops-based cluster
+
+1. From the root directory on this repo, `cd` into the `kops` directory
+2. Set up a *state* bucket for kops. This bucket will store the cluster
+[state](https://kops.sigs.k8s.io/state/).
+
+``` bash
+export KOPS_STATE_STORE=s3://<2i2c>-<hub-name>-kops-state
+aws s3 mb $KOPS_STATE_STORE --region <region>
+```
+
+```{note}
+Kops [recommends](https://kops.sigs.k8s.io/getting_started/aws/#cluster-state-storage)
+versioning the S3 bucket in case you ever need to revert or recover a previous state
+store.
+```
+
+### Create and render a kops config file
+
+You can use 
+[one](https://github.com/2i2c-org/pilot-hubs/blob/master/kops/farallon.jsonnet) of the
+existing [jsonnet](https://jsonnet.org/) specifications as a "template" for your cluster.
+You may need to tweak zones, names and instances, the rest is boilerplate to create a
+kops-based cluster accordingly to some specification already outlined in 
+[#28](https://github.com/2i2c-org/pangeo-hubs/issues/28). Once you have your jsonnet
+specification ready, you need to render it to create the config file kops understand.
+
+1. Render the `kops` config file with
+
+```bash
+jsonnet <cluster_name>.jsonnet -y > <cluster_name>.kops.yaml
+```
+
+2. Regrettably, the rendering creates yaml file with with 3 dots at the end, you can 
```

Comment:
Is this file just an unnecessary extra created by the step that has no purpose? Could you add a short sentence noting why this file is OK to be deleted?

Reply:
I am not being clear enough, I think... the generated yaml file will be used with the `kops create` command... the issue here is that jsonnet creates this file with 3 points at the very end of the file.

Reply:
Resolving since I think the text is clear enough :wink: